bigtable/client/data.h

```diff
@@ -142,6 +142,19 @@ class Table {
    */
   void Apply(SingleRowMutation&& mut);
 
+  /**
+   * Attempts to apply mutations to multiple rows.
+   *
+   * @param mut the mutations, notice that this function takes
+   * ownership (and then discards) the data in the mutation.
```

Comment:
Should the next line of the `@param` description be indented 4 lines?

Reply:
Doxygen does not care, and it is a bit more readable.  Fixed, but it will not be trivial to enforce.